corehq/apps/app_manager/views/form_designer.py

```diff
@@ -76,6 +76,19 @@ def form_designer(request, domain, app_id, module_id=None, form_id=None,
         'custom_intents': domain_has_privilege(domain, privileges.CUSTOM_INTENTS),
     })
 
+    scheduler_data_nodes = []
+    if getattr(module, 'has_schedule', False):
+        scheduler_data_nodes = [
+            "unscheduled_visit",
+            "current_visit_number",
+            "next_visit",
+            "next_visit_date",
+            "next_due",
+        ]
```

Comment:
@proteusvacuum what happens if one of these conflicts with something already in the form? should they not be namespaced in a group or something? (sorry i know these questions are unrelated to this PR, but just curious)